crmsh/bootstrap.py

```diff
@@ -1258,29 +1267,17 @@ def init_csync2():
         error("Can't create csync2 key {}".format(CSYNC2_KEY))
     status_done()
 
+    csync2_file_list_content = ""
+    for f in FILES_TO_SYNC:
+        csync2_file_list_content += "include {};\n".format(f)
+
     utils.str2file("""group ha_group
 {
 key /etc/csync2/key_hagroup;
 host %s;
-include /etc/booth;
-include /etc/corosync/corosync.conf;
-include /etc/corosync/authkey;
-include /etc/csync2/csync2.cfg;
-include /etc/csync2/key_hagroup;
-include /etc/ctdb/nodes;
-include /etc/drbd.conf;
-include /etc/drbd.d;
-include /etc/ha.d/ldirectord.cf;
-include /etc/lvm/lvm.conf;
-include /etc/multipath.conf;
-include /etc/samba/smb.conf;
-include /etc/sysconfig/nfs;
-include /etc/sysconfig/pacemaker;
-include /etc/sysconfig/sbd;
-include /etc/pacemaker/authkey;
-include /etc/modules-load.d/watchdog.conf;
+%s
```

Comment:
Maybe it would be better to use `format` here. But both work

Reply:
Changed
Thanks!